src/Microsoft.Diagnostics.Tracing.EventSource.Redist/dir.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <PackageVersion>2.0.1</PackageVersion>
-    <AssemblyVersion>2.0.1.0</AssemblyVersion>
+    <PackageVersion>2.1.0</PackageVersion>
+    <AssemblyVersion>2.0.2.0</AssemblyVersion>
```

Comment:
We would like this package and assembly to be versioned just like the rest of the packages/assemblies in CoreFX.  Given that, shoudl the package version be 2.2.0?  Also, how do you decide what the assembly version should be?

Reply:
With every release we bump the patch (3rd) portion of the assembly version.  If API changes are made, the minor (2nd) should be bumped by the dev at the time of the API change.

With every release we bump the minor (2nd) portion of the package version.  This gives us the ability to ship packages from servicing that only increment the 3rd portion of the package version.